pytorch_lightning/trainer/logging.py

```diff
@@ -38,14 +38,12 @@ def configure_logger(self, logger):
             self.logger.rank = 0
 
     def log_metrics(self, metrics, grad_norm_dic, step=None):
-        """Logs the metric dict passed in.
-
+        """Logs the metric dict passed in. If 'step' parameter is None and
+        'step' key is presented is metrics, uses metrics['step'] as a step
         :param metrics:
         :param grad_norm_dic:
+        :param step:
```

Comment:
pls, add a description and data type for all parameters

Reply:
This is a internal function, but why not. Added short docs